apps/demos/Demos/TreeView/ItemSelectionAndCustomization/jQuery/styles.css

```diff
@@ -12,7 +16,7 @@
   padding: 20px;
   margin-left: 20px;
   background-color: rgba(191, 191, 191, 0.15);
-  font-size: 115%;
+  font-size: initial;
```

Comment:
You shouldn't change font size or imagine something else. In Vue we don't have whitespace between treeview and options elements. Why in jQuery, Angular, React we have, but not in Vue?